lib/old-roman.js

```diff
@@ -0,0 +1,21 @@
+var legend = {1000: 'M',
+              500:  'D',
+              100:  'C',
+              50:   'L',
+              10:   'X',
+              5:    'V',
+              1:    'I'}
+
+function convertToOldRoman(number){
+  var workingNumber = number;
+  var Roman = '';
+  Object.keys(legend).reverse().forEach(function(key){
+    intKey = parseInt(key);
```

Comment:
what if you swapped the key/value order in your object above? would that allow for you to skip this step? or potentially order your logic below a bit differently?